server/elastic/scheduler.go

```diff
@@ -130,6 +146,8 @@ func (es *Elastic) ListNodes(ctx context.Context, req *pbs.ListNodesRequest) (*p
 	res, err := es.client.Search().
 		Index(es.nodeIndex).
 		Type("node").
+		Version(true).
+		Size(1000).
```

Comment:
Try to make unlimited

Reply:
According to this: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-from-size.html

Elasticsearch does have a max size of 10K. I could add another zero there, but returning that many records would probably hit other issues such as grpc response size limits. I think the only way forward is pagination, which I'd like to defer to another issue. We're nowhere near having 1000 nodes in production yet.

Reply:
OK sounds fine to me

Reply:
#303